src/platform/intel/ace/platform.c

```diff
@@ -101,6 +102,10 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_IPC);
 	ipc_init(sof);
 
+#if IS_ENABLED(CONFIG_LL_WATCHDOG)
+	watchdog_init();
+#endif
```

Comment:
there is no need to protect this with `CONFIG_LL_WATCHDOG` as `watchdog_init` exists but it's a noop.